From 3562a9e8b36dbe26e1c0462ea3ff4753cbd08f5f Mon Sep 17 00:00:00 2001
From: Yusuke Goda <yusuke.goda.sx@renesas.com>
Date: Wed, 8 Aug 2018 09:29:11 +0900
Subject: [PATCH 684/909] arm64: dts: h3ulcb: add H3ULCB board DT bindings

commit 1e95dd6dd2e5dbed0112b3ac77862b063ef5b73f from
git://git.kernel.org/pub/scm/linux/kernel/git/horms/renesas-bsp.git

Add H3ULCB Device tree bindings Documentation, listing it as a supported
board.

Board type name		| DDR			| eMMC
Signed-off-by: Meng Li <Meng.Li@windriver.com>
------------------------------------------------------
RTP0RC77951SKBX010SA00	| 4GiB(4 x 1 GiB)	| 8GB
RTP0RC77951SKBX010SA03	| 8GiB(4 x 2 GiB)	| 32GB

Signed-off-by: Yusuke Goda <yusuke.goda.sx@renesas.com>
---
 Documentation/devicetree/bindings/arm/shmobile.txt | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Documentation/devicetree/bindings/arm/shmobile.txt b/Documentation/devicetree/bindings/arm/shmobile.txt
index d8cf740132c6..3965ce3f3566 100644
--- a/Documentation/devicetree/bindings/arm/shmobile.txt
+++ b/Documentation/devicetree/bindings/arm/shmobile.txt
@@ -81,6 +81,7 @@ Boards:
     compatible = "renesas,gose", "renesas,r8a7793"
   - H3ULCB (R-Car Starter Kit Premier, RTP0RC7795SKBX0010SA00 (H3 ES1.1))
     H3ULCB (R-Car Starter Kit Premier, RTP0RC77951SKBX010SA00 (H3 ES2.0))
+    H3ULCB (R-Car Starter Kit Premier, RTP0RC77951SKBX010SA03 (H3 ES2.0))
     compatible = "renesas,h3ulcb", "renesas,r8a7795"
   - Henninger
     compatible = "renesas,henninger", "renesas,r8a7791"
-- 
2.17.1

